app/src/main/java/com/eventyay/organizer/core/auth/start/StartViewModel.java

```diff
@@ -102,6 +104,15 @@ public void setBaseUrl(String url, boolean shouldSetDefaultUrl) {
         sharedPreferenceModel.saveString(Constants.SHARED_PREFS_BASE_URL, baseUrl);
     }
 
+    public MutableLiveData<Boolean> getWarningDialog() {
+        warningDialogLiveData.setValue(sharedPreferenceModel.getBoolean(warningPreferenceKey,false));
```

Comment:
Didn't add to disposable

Reply:
We are not dealing Observables so adding to disposable will not be required.Thank you

Reply:
We are dealing with same type of scenario in isLoggedIn LiveData.